float32 fps
---
float32 newFPS